cs/config.go

```diff
@@ -189,11 +191,51 @@ across owners, you can unset it here.
 			fmt.Println("Saved")
 		},
 	})
+	rootCmd.AddCommand(&cobra.Command{
+		Use:   "set-base-url",
+		Short: "Set base url of the query endpoint",
```

Comment:
nit: Technically correct, but your random Joe isn't gonna use it thinking of query endpoints. Maybe something with more flavor?

> Control which GitHub instance you talk to by setting the base URL (eg: GitHub Enterprise)

Can probably omit `Long` with this too.